packages/api-graphql/__tests__/GraphQLAPI-test.ts

```diff
@@ -15,6 +15,7 @@ import { PubSub } from '@aws-amplify/pubsub';
 import { Cache } from '@aws-amplify/cache';
 import * as Observable from 'zen-observable';
 import axios, { CancelTokenStatic } from 'axios';
+import { Category, Framework } from '@aws-amplify/core/src';
```

Comment:
Here and everywhere else - I don't think you need the `/src`
```suggestion
import { Category, Framework } from '@aws-amplify/core';
```